### PR TITLE
Variable fixes

### PR DESCRIPTION
**Pull request by bashallc**
Missing " ' " for sgd input
'accuracy' not 'acc'

### CODE REVIEW

File: 08. Neural Networks Introduction - Python.ipynb [hunk 1]
```diff
@@ -75,14 +75,14 @@
...
+      "source": "# Let's compile the model\n\n###\n# REPLACE THE <addLoss> WITH 'categorical_crossentropy', <addOptimizer> WITH 'sgd', AND <addMetric> with 'accuracy' (INCLUDING THE QUOTES)\n###\nmodel.compile(loss = <addLoss>, optimizer = <addOptimizer>, metrics = [<addMetric>])\n###\n\n# Time to fit the model\nprint('Starting training')\n\n###\n# REPLACE THE <addTrainX> WITH train_X AND <addTrainY> WITH train_Y\n###\ntraining_stats = model.fit(<addTrainX>, <addTrainY>, batch_size = 1, epochs = 24, verbose = 0)\n###\n\nprint('Training finished')\nprint('Training Evaluation: loss = %0.3f, accuracy = %0.2f%%'\n      %(training_stats.history['loss'][-1], 100 * training_stats.history['accuracy'][-1]))",
       "execution_count": null,
       "outputs": []
     },
```
Comment (tottokug) [inline 08. Neural Networks Introduction - Python.ipynb:88]: #99 should be modified to acc-> accuracy as well.  ;)
```
accuracy, = graph.plot(training_stats.history['acc'],label = 'Accuracy')
```
to
```
accuracy, = graph.plot(training_stats.history['accuracy'],label = 'Accuracy')
```
